sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorAmqpConnection.java

```diff
@@ -133,18 +134,22 @@ public ServiceBusReactorAmqpConnection(String connectionId, ConnectionOptions co
      *
      * @param linkName The name of the link.
      * @param entityPath The remote address to connect to for the message broker.
+     * @param viaEntityPath The intermediate remote address to connect to for the message broker.
      * @param retryOptions Options to use when creating the link.
      *
      * @return A new or existing send link that is connected to the given {@code entityPath}.
      */
     @Override
-    public Mono<AmqpSendLink> createSendLink(String linkName, String entityPath, AmqpRetryOptions retryOptions) {
-        return createSession(entityPath).flatMap(session -> {
-            logger.verbose("Get or create producer for path: '{}'", entityPath);
+    public Mono<AmqpSendLink> createSendLink(String linkName, String entityPath, String viaEntityPath,
```

Comment:
There are two different names in this file. Is `viaEntityPath` the same as `transferEntityPath`? If so, we should fix the internal implementation to use the same name. This makes it hard to follow in the future.